README.rst

```diff
@@ -17,9 +17,9 @@ IOC Builder for EPICS and Kubernetes:
 
 ============== ==============================================================
 PyPI           ``pip install ibek``
```

Comment:
We need to make a new PyPI user for the epics-containers org and put its token in the org